### DISCUSSION TITLE
FloatingList - keyboard navigation is not working properly when implement the virtual scroll list option.

### DISCUSSION THREAD

**Discussion by Gopigovind** [floating-ui/floating-ui#3095] [React]
keyboard navigation is not working properly when implement the virtual scroll list option. 

I have more than 100 options, and loading all of them in the list takes time and causes performance issues. So, I am implementing a virtual scrolling list that loads options on demand based on the scroll position, and it works well with mouse interactions. However, keyboard navigation is not functioning properly using the Floating List with the useListNavigation component. Please correct the sample, if any mistake. 

please find the example : https://codesandbox.io/p/sandbox/charming-sunset-xsfwkl?file=%2Fsrc%2FApp.tsx%3A10%2C15

**Comment by atomiks** [comment]
`FloatingList` doesn't support virtualization as it relies on DOM index positions, you'll need to manually assign indices with virtualization - there's a demo for it

**Reply by Gopigovind** [reply]
Please share the demo URL
